lesson4/argirovatina_hometask.rb

```diff
@@ -0,0 +1,40 @@
+class LazyHash < Hash
+
+  def method_missing(method_name, *args, &block)
+    if self.keys.include? method_name.to_s
+      self[method_name.to_s] = self[method_name.to_s].call
+    else
+      self.values
```

Comment:
Are you sure that's a good result in case we don't find related method or key in the hash. No exceptions?